radar-auth/src/main/java/org/radarcns/auth/authentication/TokenValidator.java

```diff
@@ -110,39 +115,41 @@ public TokenValidator(ServerConfig config, long fetchTimeout) {
      * @throws TokenValidationException If the token can not be validated.
      */
     public RadarToken validateAccessToken(String token) throws TokenValidationException {
-        try {
-            DecodedJWT jwt = getVerifier().verify(token);
-            Set<String> claims = jwt.getClaims().keySet();
-            Set<String> missing = REQUIRED_CLAIMS.stream()
-                    .filter(c -> !claims.contains(c))
-                    .collect(Collectors.toSet());
-            if (!missing.isEmpty()) {
-                throw new TokenValidationException("The following required claims were missing "
-                        + "from the token: " + String.join(", ", missing));
+        for (JWTVerifier verifier : getVerifiers()) {
+            try {
+                DecodedJWT jwt = verifier.verify(token);
```

Comment:
`verifier.verify()` does check the algorithm and throws an `AlgorithmMismatchException` if they don't match. This exception will be caught since it is a subclass of `JWTVerificationException`. A debug message is then printed and the loop continues to try the other verifiers.